File: week 1/mario.c
#include <stdio.h>
#include <cs50.h>


int main (void) {
    int height;

    do {
        height = get_int("Height: ");
    } while(height < 1);
    int tempheight = height - 2;


    int counter = 0;
    for (int i = 0; i < height; i++) {
        for (int j = 0; j <= (tempheight); j++) {
            printf(" ");

        }
        for (int j = 0; j <= i; j++) {
            printf("#");
        }

        printf("  ");

        for (int j = 0; j <= i; j++) {
            printf("#");
        }

        printf("\n");
        tempheight -= 1;

    }


}

File: week 6/mario.py
from cs50 import get_int

height = 0

while ((height < 1) or (height > 8)):
    height = get_int("Height: ")

tempheight = height - 1

for i in range(height):
    for j in range(tempheight):
        print(" ", end='')

    for k in range(i+1):
        print("#", end='')

    print("  ", end='')

    for l in range(i+1):
        print("#", end='')

    print("\n", end='')

    tempheight -=1
